modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss

```
@@ -167,6 +167,10 @@ body .ddm-user-view-content {
 		display: none;
 		padding: 0;
 
+		.custom-control-label-text {
+			color: #6b6c7e !important;
+		}
+
```

Hi @icgsbr, avoid using `!important` property. It worked without using it in this case.
```suggestion
		.custom-control-label-text {
			color: #6b6c7e;
		}

```

---

Done!